lib/forms/items/checkbox.dot

```diff
@@ -4,6 +4,7 @@
       id="{{= it.compName }}"
       ml-bind="MLInput:{{= it.compName }}"
       {{? it.disabled }}disabled {{?}}
+      {{? it.checked }}checked {{?}}
```

Comment:
So what was the problem here, because I can't see what has changed since this code was first written?

Reply:
Generator changes were missing .